cheatsheets/Access_Control_Cheat_Sheet.md

```diff
@@ -91,7 +91,7 @@ The areas of caution while using MAC are:
 
 ## Permission Based Access Control
 
-The key concept in Permission Based Access Control is the abstraction of application actions into a set of *permissions*. A *permission* may be represented simply as a string-based name, for example, "READ". Access decisions are made by checking if the current user *has* the permission associated with the requested application action.
+The key concept in Permission Based Access Control is the abstraction of application actions into a set of *permissions*. A *permission* may be represented as a string-based name, for example, "READ". Access decisions are made by checking if the current user *has* the permission associated with the requested application action.
```

Comment:
Why was "simply" removed? This is giving an example of a simple way to name your permissions, rather than things like the completely unreadable security descriptors that Windows uses that look like:

```
D:PAI(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)(A;OICI;0x1200a9;;;BU)(A;CI;LC;;;BU)(A;CIIO;DC;;;BU) (A;OICIIO;GA;;;CO)
```